safe/impact_function/test/test_impact_function.py

```diff
@@ -759,26 +795,36 @@ def test_provenance_with_aggregation(self):
         hazard_category = definition(hazard_layer.keywords['hazard_category'])
 
         expected_provenance = {
-            'gdal_version': gdal.__version__,
-            'host_name': gethostname(),
-            'map_title': get_map_title(hazard, exposure, hazard_category),
-            'map_legend_title': exposure['layer_legend_title'],
-            'user': getpass.getuser(),
-            'os': readable_os_version(),
-            'pyqt_version': PYQT_VERSION_STR,
-            'qgis_version': QGis.QGIS_VERSION,
-            'qt_version': QT_VERSION_STR,
-            'inasafe_version': get_version(),
-            'aggregation_layer': aggregation_layer.source(),
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->